zerver/webhooks/stripe/view.py

```diff
@@ -14,10 +14,9 @@
 
 @api_key_only_webhook_view('Stripe')
 @has_request_variables
-def api_stripe_webhook(request, user_profile,
-                       payload=REQ(argument_type='body'), stream=REQ(default='test'),
-                       topic=REQ(default=None)):
-    # type: (HttpRequest, UserProfile, Dict[str, Any], Text, Optional[Text]) -> HttpResponse
+def api_stripe_webhook(request: HttpRequest, user_profile: UserProfile,
+                       payload: Dict[str, Any]=REQ(argument_type='body'), stream: Text=REQ(default='test'),
```

Comment:
I appreciate this has the same layout as the original, but given the styling of the other changes, my personal preference would be to move the `stream` parameter to the next line.